webpack.config.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
   externals: [
     // Treat all sdk requires as importable commonjs modules.
     (context, request, callback) => {
-      if (/^sdk\//.test(request)) {
+      if (/^sdk\//.test(request) || /^resource\:\/\//.test(request)) {
```

Comment:
Totally true, though I'd rather keep them explicitly separate for readability purposes.